grails-app/views/comment/index.gsp

```diff
@@ -0,0 +1,29 @@
+<%--
```

Comment:
Is this page ever used, or are comments only intended to be embedded within other pages? If index is never used, please remove it.